pkg/make/builder.go

```diff
@@ -441,7 +442,7 @@ func (b *Builder) execBindata(ctx context.Context, dir string) error {
 	return nil
 }
 
-func bootstrapFromNotary(notaryConfigDir, remoteServerURL, localRepo, gun string) error {
+func bootstrapFromNotary(notaryConfigDir, remoteServerURL, localRepo, gun string, retryTimeout time.Duration, bootstrapTimeout time.Duration) error {
```

Comment:
I'd hard code the values instead of passing. But whatev

Reply:
I did this so the test wouldn't take 5min to run